docs/securing_fledge.rst

```diff
@@ -159,6 +171,15 @@ To add a new user from the *User Management* page select the *Add User* icon in
 
 You can select a role for the new user, a user name and an initial password for the user. Only users with the role *admin* can add new users.
 
+Update User Details
+-------------------
+
+The edit user option allows the name, authentication method and description of a user to be updated.
```

Comment:
This option is only available for `admin` role types.

And the User with the other roles can update their profile ( only allowed fields are - `Name, Password`)